pubsub/system-test/subscriptions.test.js

```diff
@@ -13,128 +13,139 @@
 
-  describe('createSubscription', function () {
-    it('should create a subscription', function (done) {
-      program.createSubscription(topicNameOne, subscriptionNameOne, function (err, subscription, apiResponse) {
-        assert.ifError(err);
-        assert.equal(subscription.name, fullSubscriptionNameOne);
-        assert(console.log.calledWith('Created subscription %s to topic %s', subscriptionNameOne, topicNameOne));
-        assert.notEqual(apiResponse, undefined);
-        // Listing is eventually consistent, so give the index time to update
-        setTimeout(done, 5000);
-      });
+  it(`should create a push subscription`, (done) => {
+    const output = run(`${cmd} create-push ${topicName} ${subscriptionNameTwo}`, cwd);
+    assert.equal(output, `Subscription ${fullSubscriptionNameTwo} created.`);
+    pubsub.subscription(subscriptionNameTwo).exists((err, exists) => {
+      assert.ifError(err);
+      assert.equal(exists, true);
+      done();
     });
   });
 
-  describe('getSubscriptionMetadata', function () {
-    it('should get metadata for a subscription', function (done) {
-      program.getSubscriptionMetadata(subscriptionNameOne, function (err, metadata) {
-        assert.ifError(err);
-        assert.equal(metadata.name, fullSubscriptionNameOne);
-        assert(console.log.calledWith('Got metadata for subscription: %s', subscriptionNameOne));
-        done();
-      });
-    });
+  it(`should get metadata for a subscription`, () => {
+    const output = run(`${cmd} get ${subscriptionNameOne}`, cwd);
+    const expected = `Subscription: ${fullSubscriptionNameOne}` +
+      `\nTopic: ${fullTopicName}` +
+      `\nPush config: ` +
+      `\nAck deadline: 10s`;
+    assert.equal(output, expected);
   });
 
-  describe('listSubscriptions', function () {
-    it('should list subscriptions', function (done) {
-      program.listSubscriptions(topicNameOne, function (err, subscriptions) {
-        assert.ifError(err);
-        assert(Array.isArray(subscriptions));
-        assert(subscriptions.length > 0);
-        var recentlyCreatedSubscriptions = subscriptions.filter(function (subscription) {
-          return subscription.name === fullSubscriptionNameOne || subscription.name === fullSubscriptionNameTwo;
-        });
-        assert.equal(recentlyCreatedSubscriptions.length, 1, 'list only has one newly created subscription');
-        assert.equal(recentlyCreatedSubscriptions[0].name, fullSubscriptionNameOne, 'list has correct newly created subscription');
-        assert(console.log.calledWith('Found %d subscription(s)!', subscriptions.length));
-        done();
-      });
-    });
+  it(`should list all subscriptions`, (done) => {
+    // Listing is eventually consistent. Give the indexes time to update.
+    setTimeout(() => {
+      const output = run(`${cmd} list`, cwd);
+      assert.notEqual(output.indexOf(`Subscriptions:`), -1);
+      assert.notEqual(output.indexOf(fullSubscriptionNameOne), -1);
+      done();
```

Comment:
Does `fullSubscriptionNameTwo` show up here - and if it does, should we check for it?